<!DOCTYPE html><html><head><meta charSet="utf-8"/><meta name="viewport" content="width=device-width,initial-scale=1"/><meta name="google-site-verification" content="c8xLy6CbRSnA2A3H8CUama6rq8hjtG1c_lBvFhO_nYs"/><title>Keynote Speakers</title><meta name="description" content=""/><meta property="og:title" content="ISBRA 2022 - Keynote Speakers"/><meta property="og:description" content=""/><meta name="next-head-count" content="7"/><link rel="preload" href="ad03d55cb9cb1936.css" as="style"/><link rel="stylesheet" href="ad03d55cb9cb1936.css" data-n-g=""/><link rel="preload" href="cdd435a6ec948ad9.css" as="style"/><link rel="stylesheet" href="cdd435a6ec948ad9.css" data-n-p=""/><noscript data-n-css=""></noscript><script defer="" nomodule="" src="polyfills-5cd94c89d3acac5f.js"></script><script src="webpack-6f45cb54dcc402af.js" defer=""></script><script src="framework-a87821de553db91d.js" defer=""></script><script src="main-803fd245650d2bfd.js" defer=""></script><script src="_app-36a30e3cbeb6cb10.js" defer=""></script><script src="keynotes-86ffc9000ba872ec.js" defer=""></script><script src="_buildManifest.js" defer=""></script><script src="_ssgManifest.js" defer=""></script><script src="_middlewareManifest.js" defer=""></script></head><body><div id="__next"><header class="Layout_header__KD6xC"><div><h2>EEBG 2023</h2></div></header><section class="Layout_hero__dJiuI"><div><h2>East European Bioinformatics and Computational Genomics School (EEBG)<p></p> June 26 - July 7, 2023<!-- --><div></div></h2></div></section><div class="Layout_root__XpAuI"><nav class="Layout_menu__N__qO"><h2>Menu</h2><ul><li><a target="_self" href="https://mangul-lab-usc.github.io/EEBG2023/">Home</a></li><li><a target="_self" href="https://mangul-lab-usc.github.io/EEBG2023/dates">Important Dates</a></li><li><a target="_self" href="https://mangul-lab-usc.github.io/EEBG2023/registration">Registration</a></li><li><a target="_self" href="https://mangul-lab-usc.github.io/EEBG2023/agenda">Program Agenda</a></li><li><a target="_self" href="https://mangul-lab-usc.github.io/EEBG2023/keynotes">Guest and keynote speakers</a></li><li><a target="_self" href="https://mangul-lab-usc.github.io/EEBG2023/committees">Committees</a></li><li><a target="_self" href="https://mangul-lab-usc.github.io/EEBG2023/previous-meetings">Previous Meetings</a></li></ul></nav><div class="Layout_stripe__65gNv"></div><main class="Layout_main__gbdP0"><h1>Keynote Speakers </h1>
<style>
.hidden {
  background-color: #f7f5f5;
  border: 2px solid #f7f5f5;
  padding: 10px;
  border-radius: 1rem;  
}
</style>
<div class="hidden"><div><img src="alser.jpg" style = "border-radius: 8px; float: left; margin-right: 15px;" width="200" height="250"/><h3>Mohammed Alser</h3><span>ETH Zürich</span></div><div><p>Mohammed Alser is a Senior Researcher and Lecturer of bioinformatics and computer architecture at ETH Zurich since September 2018. His primary research incorporates several aspects of bioinformatics, metagenomics, computational genomics, and computer architecture. He is particularly interested in rethinking the complete compute stack, starting from the way we handle input data and algorithms, to the underlying hardware architecture for enabling intelligent genomic analyses on existing and unconventional (e.g., non-von Neumann) computing technologies. He focuses on incorporating intelligent genomic analyses at population scale into clinical practice for rapid surveillance of disease outbreaks, diagnosis of genetic disorders, and identification of pathogens and microbiomes around us. He is a member of the BioPIM project of the European Innovation Council’s Pathfinder program for accelerating genome analysis. He has co-authored a healthy number of top-tier research papers. He earned his PhD in Computer Engineering from Bilkent University in August 2018. He received the ETH Zürich Exceptional Performance Award for two consecutive years, IEEE Turkey Doctoral Dissertation Award, the Yasser Arafat Award for the Best Palestinian PhD Student in Turkey, the prestigious TÜBITAK doctoral fellowship, and the HiPEAC Collaboration Grant.</p></div></div><div>&nbsp</div>
<div class="hidden"><div><img src="alkan.jpg" style = "border-radius: 8px; float: left; margin-right: 15px;" width="285" height="250"/><h3>Can Alkan</h3><span>Bilkent University
</span></div><p>Can Alkan is currently an Associate Professor at the Department of Computer Engineering at Bilkent University since January 2012. He graduated from Bilkent University Dept. of Computer Engineering in 2000, and received his Ph.D. in Computer Science from Case Western Reserve University in 2005 after a brief visit to Simon Fraser University. During his Ph.D. he worked on the evolution of centromeric DNA, RNA-RNA interaction prediction and RNA folding problems. He then joined the Department of Genome Sciences of the University of Washington as a postdoctoral fellow. Since then his work includes computational prediction of human genomic structural variation, and characterization of segmental duplications and copy-number polymorphisms using high throughput sequencing data, and acceleration of genome analysis through hardware/software co-development.
</p></div><div>&nbsp</div>
<div class="hidden">
<div><img src="enc.png" style = "border-radius: 8px; float: left; margin-right: 15px;" width="200" height="250"/><h3>Encarnita Mariotti-Ferrandiz</h3><span>Sorbonne Université
</span></div><p>Encarnita Mariotti-Ferrandiz is an Associate Professor in Immunology from Sorbonne Université, working in the i3 lab directed by Prof. David Klatzmann, located at Pitié-Salpêtrière Hospital, in Paris. Since her PhD, she has been involved in deciphering the Adaptive Immune Receptor Repertoire (AIRR) diversity in health and pathology with a major focus on T-cell biology. From sample to statistical modelling, she is interested in (i) refining T cell differentiation and selection knowledge in health and disease and (ii) identifying biomarkers of diseases. Her main field of research in immunology focus on the study of autoimmune in inflammatory diseases. T-cells and the T-cell receptor being a part of a whole in T-cell biology, she uses the Systems Immunology approach to better characterize the immune system by coordinating the integration of multi-scale biological (deep cell phenotype, transcriptome, repertoire, microbiome) and clinical information. As an Associate Professor, she launched the “Integrative and Systems Immunology” Master 2 curriculum at Sorbonne Université in 2019, with the aim to promote and train the future research generation to Systems Immunology as a whole, including AIRR.
</p></div>
<div>&nbsp</div>
<div class="hidden">
<div><img src="mark.png" style = "border-radius: 8px; float: left; margin-right: 15px;" width="200" height="200"/><h3>Mark Chaisson</h3><span>University of Southern California</span></div><p>Mark Chaisson is an assistant professor at the Department of Computational Biology and Bioinformatics, University of Southern California. His research primarily focuses on the development of new methods for genetic analysis using novel sequencing types, with a specific emphasis on de novo assembly and the detection of structural variation (SV). Dr. Chaisson's expertise in computer science and bioinformatics was developed during his training with Dr. Pavel Pevzner at the University of California, San Diego. He also received training in genetics from Dr. Evan Eichler at the University of Washington. During his PhD studies, Dr. Chaisson developed the first de Bruijn assembler for short-read data, which became the dominant paradigm for short-read assembly. Following this, he worked at Pacific Biosciences, where he developed and published the first method for rapidly aligning long, noisy single-molecule sequencing (SMS) reads. As a postdoctoral scholar, Dr. Chaisson's work focused on the detection of human SV using SMS, which demonstrated that an order of magnitude more variation may be detected using this approach than with typical short-read-based studies. Dr. Chaisson's research in SV has continued with the Human Genome Structural Variation Consortium (HGSVC), where he led research cataloging SV in three trios of diverse descent using a set of six diverse sequencing platforms. This consortium comprises over 80 researchers worldwide, and Dr. Chaisson's role in organizing such an effort makes him uniquely suited to continue this research while starting his independent research program at the University of Southern California.</p></div>
<div>&nbsp</div>
<div class="hidden">
<div><img src="ax.jpg" style = "border-radius: 8px; float: left; margin-right: 15px;" width="200" height="200"/><h3>Andrada Ficustean</h3><span>Massachusetts Institute of Technology</span></div><p>Andrada Fiscutean is a science and technology journalist based in Bucharest, Romania. She’s a former Knight Science Journalism Fellow at MIT and has written for Ars Technica, Nature, The Guardian, Foreign Policy, Motherboard, CSO Online & more. Andrada is also editor-in-chief of ProFM radio in Romania and teaches science journalism at the University of Bucharest.</p>
</div>
<div>&nbsp</div>
<div class="hidden">
<div><img src="serghei-mangul.png" style = "border-radius: 8px; float: left; margin-right: 15px;" width="200" height="200"/><h3>Serghei Mangul</h3><span>University of Southern California, USA</span></div><p>Dr. Mangul is an Assistant Professor of Clinical Pharmacy and Computational Biology at the University of Southern California. He specializes in the design, development, and application of novel data-driven computational approaches to accelerate the diffusion of genomics and biomedical data into translational research and education. Dr. Mangul is a passionate advocate for promoting transparency and reproducibility in data-driven biomedical research, as well as for making bioinformatics education accessible to all. Dr. Mangul’s work is dedicated to advancing the principles of reproducibility, data sharing, and software usability, with the ultimate goal of shaping a more equitable and impactful future for the field of bioinformatics. Dr. Mangul received his Ph.D. in Bioinformatics from Georgia State University, and he holds a B.Sc. in Applied Mathematics from Moldova State University, Chisinau, Moldova. He completed his postdoctoral training in computational genomics with Prof. Eskin at the University of California Los Angeles (UCLA). Dr. Mangul is the recipient of the prestigious National Science Foundation CAREER award, and he serves as a mentor for the Inaugural National Institutes of Health (NIH) cohort of the Artificial Intelligence/Machine Learning Consortium to Advance Health Equity and Researcher Diversity (AIM-AHEAD) Fellowship Program in Leadership.</p></div>
<div>&nbsp</div>
<div class="hidden">
<div><img src="nicole.jpg" style = "border-radius: 8px; float: left; margin-right: 15px;" width="200" height="250"/><h3>Nicole Nogoy</h3><span>Executive Editor, GigaScience</span></div><p>Nicole is the Executive Editor of GigaScience having joined the team in 2012. She is a keen advocate of Open Science, Open Data and reproducible research. She has a PhD in Natural Sciences (Cardiovascular Molecular Medicine) from the University of Goettingen, Germany and obtained her BSc Hons in Biochemistry & Molecular Biology and Physiology from Victoria University of Wellington, New Zealand. Nicole has over a decade of experience in the STM publishing industry – most notably having launched and been Managing Editor of Genome Medicine.</p></div>
<div>&nbsp</div>
<div class="hidden">
<div><img src="pawel.jpg" style = "border-radius: 8px; float: left; margin-right: 15px;" width="250" height="250"/><h3>Pawel P. Labaj</h3><span>Jagiellonian University
</span></div><p>Dr. Pawel P. Labaj, MS, DSc, Ph.D. is an Bioinformatics Group Leader at the Malopolska Center of Biotechnology of the Jagiellonian University. He completed his Doctor of Philosophy at the University of Natural Resources and Life Sciences and Master of Science at the Silesian University of Technology. His clinical interests are Pattern Recognition, Computational Biology, Genomics, Computational Biology, Gene Expression, Next Generation Sequencing, Transcriptomics, Data Mining, Bioinformatics, and Biotechnology.</p></div>
<div>&nbsp</div>
<div class="hidden">
<div><img src="shilpa.jpg" style = "border-radius: 8px; float: left; margin-right: 15px;" width="310" height="250"/><h3>Shilpa Garg</h3><span>Novo Nordisk Foundation Center for Biosustainability
Technical University of Denmark
</span></div><p>Shilpa Garg is a Senior Researcher/Group Leader/Associate Professor at the Technical University of Denmark. She is passionate to apply the power of computers to understand the biology of complex disease and sustainability analyses. In her talk, Prof. Garg will present an efficient algorithm for performing accurate haplotype-resolved assembly of human individuals. This algorithm takes advantage of new long accurate data types such as PacBio HiFi and long-range Hi-C data. Additionally, she will discuss a generalized graph-based method for phased assembly of cancer genomics, which has produced the first precise somatic and germline structural variant landscape required for better drug therapeutics. She has published in prestigious journals like Bioinformatics, Nature Communications, and Nature Biotechnology.
</p></div>
<div>&nbsp</div>
<div class="hidden">
<div><img src="taras-oleksyk.jpg" style = "border-radius: 8px; float: left; margin-right: 15px;" width="200" height="250"/><h3>Taras Oleksyk</h3><span>Oakland University, USA</span></div><p>Dr. Taras Oleksyk is an Assistant Professor at the Oakland University. Dr. Oleksyk research interests are in evolution, comparative genomics, environmental toxicology, and genetic epidemiology. His background is in genomics, specifically as it relates to adaptation, speciation and disease. His scientific interests are in two general areas. The first is in genomics and admixture of human populations. His lab is involved in several studies focusing on human population genetics and the analysis of the candidate disease genes in the Caribbean and Eastern Europe. His other interest is in evolution and conservation of endangered species in the Caribbean. Dr. Oleksyk is involved in studies focusing on genome assembly and analysis; selection scans (Amazon parrots, Hispaniolan solenodons, African cheetah and elephants). Generally, he is interested in the study of genome diversity and its implications to evolutionary processes of adaptation and speciation.</p>
</div>
<div>&nbsp</div>
<div class="hidden">
<div><img src="robinson.png" style = "border-radius: 8px; float: left; margin-right: 15px;" width="150" height="200"/><h3>Mark D Robinson</h3><span>University of Zurich</span></div><p>Mark Robinson has been an Associate Professor since 2017 after joining the Department of Molecular Life Sciences of the University of Zurich (UZH) in 2011. He studied Applied Mathematics (BSc, Uni. Guelph) and Statistics (MSc, Uni. British Columbia), and did a PhD in statistical bioinformatics at the University of Melbourne. He has predoctoral experience at the Banting and Best Department of Medical Research (Uni Toronto) and postdoctoral experience in Cancer Epigenomics at the Garvan Institute in Sydney. The Robinson group at UZH develops statistical methods for interpreting high-throughput sequencing and other genomics technologies in the context of genome sequencing, gene expression and regulation and analysis of epigenomes, with a current focus on the analysis of single-cell and spatial datasets.</p></div>
<div>&nbsp</div>
<div class="hidden">
<div><img src="alex-zelikovsky.jpg" style = "border-radius: 8px; float: left; margin-right: 15px;" width="250" height="250"/><h3>Alexander Zelikovsky</h3><span>Georgia State University, USA</span></div><p>Dr. Alexander Zelikovsky is a Distinguished University Professor at the Department of Computer Science at Georgia State University, USA. Prof. Zelikovsky is known for an approximation algorithm for the minimum Steiner tree problem with the approximation ratio 1.55. Other areas of interest of Prof. Zelikovsky are bioinformatics; discrete algorithms and combinatorial optimization; computer-aided design for very large system integration; physical design & design for manufacturing; computational biology; graph theory; computational geometry.
</p></div>
